CIP-governance-metadata-cc-rationale/README.md

```diff
@@ -0,0 +1,133 @@
+---
+CIP: ?
+Title: Governance metadata - Constitutional Committee vote rationale
+Category: Metadata
+Status: Proposed
+Authors:
+    - Ryan Williams <ryan.williams@intersectmbo.org>
+    - Eystein Magnus Hansen <eysteinsofus@gmail.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-07-17
+License: CC-BY-4.0
+---
+
+## Abstract
+
+The Conway ledger era ushers in on-chain governance for Cardano via [CIP-1694 | A First Step Towards On-Chain Decentralized Governance](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md), with the addition of many new on-chain governance artifacts.
+Some of these artifacts support the linking off-chain metadata, as a way to provide context.
+
+The [CIP-100 | Governance Metadata](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0100) standard provides a base framework for how all off-chain governance metadata should be formed and handled.
+But this is intentionally limited in scope, so that it can be expanded upon by more specific subsequent CIPs.
+
+This proposal aims to provide a specification for off-chain metadata vocabulary that can be used to give context to Constitutional Committee votes.
```

Comment:
Article VI Section 4 of the Interim Constitution:

>Constitutional Committee processes shall be transparent. The Constitutional Committee shall publish each decision. When voting no on a proposal, the Committee shall set forth the basis for its decision with reference to specific Articles of this Constitution that are in conflict with a given proposal.

Does this mean that we should explicitly mention that the vote rationale is only expected if an ICC or CC member votes **NO** at a later stage?

Reply:
This is a good point
I think we can add this to the motivation of the proposal.

Reply:
Yeah good catch. It does not exclude giving a rationale when voting yes, nor is it specifically saying that the rational needs to follow this metadata but I think its a worthy mention for the motivation of this proposal.

Reply:
Have added a note for this, thanks